modules/logger/parser/parser.go

```diff
@@ -0,0 +1,185 @@
+// Package logger/parser contains methods to parse and restructure log output from go testing and terratest
+package parser
+
+import (
+	"bufio"
+	"io"
+	"regexp"
+	"strings"
+	"sync"
+
+	junitparser "github.com/jstemmer/go-junit-report/parser"
+	"github.com/sirupsen/logrus"
+)
+
+// SpawnParsers will spawn the log parser and junit report parsers off of a single reader.
+func SpawnParsers(logger *logrus.Logger, reader io.Reader, outputDir string) {
+	pr, pw := io.Pipe()
```

Comment:
It's hard to guess what one and two letter variable names stand for. Longer, more descriptive names typically make the code easier to read & maintain.

Reply:
fixed